openstack-mixin/dashboards.libsonnet

```diff
@@ -0,0 +1,140 @@
+    local stat = g.panel.stat;
+    {
+      cloudOverview:
+        g.dashboard.new('OpenStack cloud overview')
+        + g.dashboard.withPanels(
+          g.util.grid.wrapPanels(
+            [
+              panels.placementStatus { gridPos+: { w: 4 } },
+              panels.keystoneStatus { gridPos+: { w: 4 } },
+              panels.novaStatus { gridPos+: { w: 4 } },
+              panels.neutronStatus { gridPos+: { w: 4 } },
+              panels.cinderStatus { gridPos+: { w: 4 } },
+              panels.glanceStatus { gridPos+: { w: 4 } },
+              panels.alertsPanel { gridPos+: { w: 8 } },
+              panels.totalResources { gridPos+: { w: 16 } },
+              g.panel.row.new('Keystone service'),
+              panels.domains { gridPos+: { w: 4 } },
+              panels.projects { gridPos+: { w: 4 } },
+              panels.regions { gridPos+: { w: 4 } },
+              panels.users,
+              panels.projectDetails { gridPos+: { w: 24 } },
+            ], 12, 8
+          )
+        )
+        // hide link to self
+        + root.applyCommon(vars.multiInstance, uid + '-cloud-overview', tags, links { cloudOverview+:: {} }, timezone, refresh, period),
+      novaNeutronOverview:
+        g.dashboard.new('OpenStack nova and neutron overview')
+        + g.dashboard.withPanels(
+          g.util.grid.wrapPanels(
+            [
+              g.panel.row.new('Nova service'),
+              panels.novaStatus { gridPos+: { w: 6 } },
+              panels.vms { gridPos+: { w: 18 } },
+              panels.instanceUsage,
+              panels.vCPUUsage,
+              panels.memoryUsage,
+              panels.novaAgents,
+              g.panel.row.new('Neutron service'),
+              panels.neutronStatus { gridPos+: { w: 6 } },
+              panels.networks { gridPos+: { w: 9 } },
+              panels.subnets { gridPos+: { w: 9 } },
+              panels.routers,
+              panels.routerDetails,
+              panels.ports,
+              panels.portDetails,
+              panels.floatingIPs,
+              panels.ipsUsed,
+              panels.securityGroups,
+              panels.neutronAgents,
+            ], 12, 8
+          )
+        )
+        // hide link to self
+        + root.applyCommon(vars.multiInstance, uid + '-nova-and-neutron-overview', tags, links { novaNeutronOverview+:: {} }, timezone, refresh, period),
+      cinderGlanceOverview:
+        g.dashboard.new('OpenStack cinder and glance overview')
+        + g.dashboard.withPanels(
+          g.util.grid.wrapPanels(
+            [
+              g.panel.row.new('Cinder service'),
+              panels.cinderStatus { gridPos+: { w: 4 } },
+              panels.volumes { gridPos+: { w: 10 } },
+              panels.volumeStatus { gridPos+: { w: 10 } },
+              panels.volumeUsage,
+              panels.backupUsage,
+              panels.poolUsage,
+              panels.cinderAgents,
+              g.panel.row.new('Glance service'),
```

Comment:
Would `Glance services` be better here?

Reply:
Glance is the name of the service in this case